### PR TITLE
Fix compilation of example

### PR DESCRIPTION
**Pull request by Inkrementator**
JSON5 token types are ignored, this might be undesirable.

### CODE REVIEW

File: examples/formatjson/formatjson.d
Comment (schveiguy) [inline examples/formatjson/formatjson.d:88]: These should not ever happen, they should be an assert(0).

JSON5 tokens are not returned if you are not using JSON5 mode. Spaces could be included, but again, this code isn't selecting for them.

I have mixed feelings about including JSON5 tokens in the main enum, but it's so much cleaner than templatizing, and ugh, that gets messy.

File: examples/formatjson/formatjson.d
Comment (schveiguy) [inline examples/formatjson/formatjson.d:12]: I was going to say this needs the `false` equivalent for replacing escapes, but I'm realizing I changed the default!

Ah well, I think it's better this way anyway.

**Review by schveiguy — CHANGES_REQUESTED** [review]
Thank you for taking the time to look at this!

File: examples/formatjson/formatjson.d
Comment (Inkrementator) [reply examples/formatjson/formatjson.d]: I think these should be publically imported by the main package. I'll make a pull request for io too.

Comment (schveiguy) [reply examples/formatjson/formatjson.d]: I somewhat agree. though it's going to conflict with phobos...

**Review by schveiguy — APPROVED** [review]

File: examples/formatjson/formatjson.d
Comment (Inkrementator) [reply examples/formatjson/formatjson.d]: I just saw that you're maintainer of io too, so I won't make a PR for this 1 line change. Your call if this breaking change is worth it.

Comment (schveiguy) [reply examples/formatjson/formatjson.d]: I have big plans to change io so it has async support. So it's not something that's high on my priority list.

**Pull request merged**
